templates/al2023/provisioners/install-nodeadm.sh

```diff
@@ -13,6 +13,10 @@ sudo nerdctl run \
   public.ecr.aws/eks-distro-build-tooling/golang:1.21 \
   make build
 
+# cleanup images and networks used for nerdctl
```

Comment:
do we need to add corresponding test/verification after AMI build?

Reply:
can definitely do this, let me see if i can setup a test

Reply:
what test would be most effective? not 100% sure about network guarantees, like checking interface names/count. the existence of `nerdctl0` is crude but works i guess. any thoughts?

Reply:
I think at build AMI level, check the existence of nerdctl0 would be a good start point